src/AttachmentPreview/AttachmentPreview.less

```diff
@@ -0,0 +1,195 @@
+@import (reference) "../less/utilities";
+
+@HEADER_BAR_HEIGHT: @size_5xl;
+@DESKTOP_BUFFER_SIZE: @size_l;
+@MOBILE_BUFFER_SIZE: @size_xs;
+
+// TODO: can I add this to sizing.less?
+@size_ml: 1.25rem;
+
+.AttachmentPreview--Background {
+  .zIndex--5;
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  background: fade(@neutral_black, 60%);
+}
+
+.AttachmentPreview--HeaderBar {
+  .zIndex--6;
+  background-color: @neutral_black;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: @HEADER_BAR_HEIGHT;
+  .alignItems(center);
+  color: @neutral_white;
+}
+
+.AttachmentPreview--AttachmentName {
+  line-height: @size_l;
+  // TODO: why isn't this adding the ellipsis?
+  .text--truncate();
+}
+
+.AttachmentPreview--FileIcon {
+  max-height: @size_l;
+  max-width: @size_l;
+  padding: @size_ml @size_s @size_ml @size_l;
+  .padding--left--l();
+  filter: brightness(0) invert(1); /* makes the file icon white without needing a different <img> src attr */
+}
+
+.AttachmentPreview--PreviewWindow {
+  .zIndex--6;
+  position: fixed;
+  top: calc(@HEADER_BAR_HEIGHT + @DESKTOP_BUFFER_SIZE);
+  left: 50%;
+  transform: translateX(-50%);
+  height: calc(100vh - @HEADER_BAR_HEIGHT - (2 * @DESKTOP_BUFFER_SIZE));
+  width: calc(100vw - (2 * @DESKTOP_BUFFER_SIZE));
+}
+
+.AttachmentPreview--ImageContainer {
+  .zIndex--7;
+  margin: 0 auto;
+  height: 100%;
+  width: 100%;
+  .alignItems(center);
+  .justifyContent(center);
+}
+
+.AttachmentPreview--PreviewWindow img {
+  .zIndex--8;
+  // render image as larger as natural size, or scale down to fit screen
+  object-fit: scale-down;
+  height: 100%;
+  width: 100%;
+
+  // TODO: fix this
```

Comment:
I know this is annoying, but any chance you could make a JIRA ticket for this and then perhaps link the ticket # (e.g. `M5G-123`) inline with this TODO?